packages/valory/connections/p2p_libp2p_client/connection.yaml

```diff
@@ -47,4 +47,4 @@ dependencies:
     version: <1.5.0,>=1.4.0
   open-aea-ledger-cosmos: {}
   open-aea-ledger-ethereum: {}
-is_abstract: false
+is_abstract: true
```

Comment:
That is not ideal. The flag should be set as an override for each agent using it (i.e. in `aea-config`). By having it default as abstract it will be confusing when using the connection in different context.

Reply:
Done in 470747bdaa24f5649f2af4aa2ee01e3fcbf5361b. I tried to avoid verbose configs for all the agents, but yes, it makes more sense that way.